feature-runner/steps/device/connectDevice.ts

```diff
@@ -29,20 +29,34 @@ export const connectDevice = async ({
 		certsDir,
 		id: intermediateCertId,
 	})
-	const [deviceKey, deviceCert, intermediateCA, digiCert, baltimore] =
-		await Promise.all([
-			fs.readFile(deviceFiles.privateKey, 'utf-8'),
-			fs.readFile(deviceFiles.cert, 'utf-8'),
-			fs.readFile(intermediateCAFiles.cert, 'utf-8'),
-			fs.readFile(
-				path.join(process.cwd(), 'data', 'DigiCertTLSECCP384RootG5.crt.pem'),
-				'utf-8',
+	const [
+		deviceKey,
+		deviceCert,
+		intermediateCA,
+		digiCert,
+		baltimore,
+		microsoftRSARoot,
```

Comment:
> Since we are not migrating devices, we should only need the new DigiCertGlobalRootG2.pem, not the other two.

According to MS docs, they said `The Microsoft RSA Root Certificate Authority 2017 helps prevent future disruptions in case the DigiCert Global Root G2 is retired unexpectedly.`

What do you think?

Reply:
That's not relevant for the software-based device used in the end-to-end test runner.